src/utils/ethUtils.js

```diff
@@ -2,7 +2,7 @@ const getInstanceIdentifier = () => {
   const hostName = window.location.hostname;
   if (hostName === "localhost") return "demo";
 
-  return hostName.split(".")[0];
+  return hostName.split(".")[0].replace("test-", "");
```

Comment:
the hostname is `waterledger` in production, so this will just return `waterledger`. I think I haven't understand the purpose of this. Can we have a debrief?

Reply:
The hostname is actually *.waterledger.com, so this code will get the first part of the hostname as the scheme identifier. In this case "demo.waterledger.com" will return "demo" as the scheme. I have added the replace "test-" to the code to also handle "test-demo.waterledger.com" as the "demo" scheme.

Reply:
Makes sense, `test-mdwss.waterledger.com`, `test-ivrs.waterledger.com`, etc. will be the test environments.